milestone2.ts

```diff
@@ -150,11 +170,12 @@ let updateVolunteer: UpdateVolunteer; // Code here
 
 type GetVolunteerInfo = (vol: Volunteer) => String;
 let getVolunteerInfo: GetVolunteerInfo = (vol) => {
-  let name, age, city; // Code here
+  // declare variables for name, age, and city using object destructuring
+  let {name, age, city} = {...vol};
```

Comment:
Spread isn't necessary here
```suggestion
  let {name, age, city} = {vol};
```